lib/error_tracker/schemas/error.ex

```diff
@@ -22,6 +22,7 @@ defmodule ErrorTracker.Error do
     field :status, Ecto.Enum, values: [:resolved, :unresolved], default: :unresolved
     field :fingerprint, :binary
     field :last_occurrence_at, :utc_datetime_usec
+    field :muted, :boolean, default: false
```

Comment:
I would not set the default, rather than leaving it in the schema migration. 

It is a personal preference (and if you want to leave it as it is I'm not opposing to it), but with a default value you will see that getting a partial struct from a query with a `select` can cause to receivbe a `false`on this field that may not be the real value in DB.

Feel free to change it or leave it as it is, no problem :D

Reply:
Agreed. I've removed the default from the codebase and left it on the database only.